I'm getting this error locally, all the other tests pass:

```
========================================================================================== FAILURES ===========================================================================================
_____________________________________________________________________________________ test_nutpie_method ______________________________________________________________________________________

data_random_n100 =     continuous1  continuous2  continuous3  binary_num binary_cat  count1  count2 categorical1 categorical2
0     -0.81...7101    -0.583112    -0.919001           0          b       3       4            P            K

[100 rows x 9 columns]

    @pytest.mark.skipif(not NUTPIE_AVAILABLE, reason="nutpie not available")
    def test_nutpie_method(data_random_n100):
        """Test nutpie method runs successfully."""
        model = bmb.Model("continuous1 ~ continuous2", data_random_n100)
>       result = model.fit(inference_method="nutpie", draws=50, tune=50)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/test_inference_methods.py:51: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
bambi/models.py:350: in fit
    return self.backend.run(
bambi/backend/pymc.py:153: in run
    result = self._run_mcmc(
bambi/backend/pymc.py:224: in _run_mcmc
    idata = pm.sample(
.pixi/envs/dev/lib/python3.12/site-packages/pymc/sampling/mcmc.py:832: in sample
    return _sample_external_nuts(
.pixi/envs/dev/lib/python3.12/site-packages/pymc/sampling/mcmc.py:371: in _sample_external_nuts
    compiled_model = nutpie.compile_pymc_model(
.pixi/envs/dev/lib/python3.12/site-packages/nutpie/compile_pymc.py:532: in compile_pymc_model
    return _compile_pymc_model_numba(
.pixi/envs/dev/lib/python3.12/site-packages/nutpie/compile_pymc.py:241: in _compile_pymc_model_numba
    ) = _make_functions(
.pixi/envs/dev/lib/python3.12/site-packages/nutpie/compile_pymc.py:654: in _make_functions
    shapes = _compute_shapes(model)
             ^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

model = <pymc.model.core.Model object at 0x7ad5782f8cb0>

    def _compute_shapes(model) -> dict[str, tuple[int, ...]]:
        import pytensor
        from pymc.initial_point import make_initial_point_fn
    
        point = make_initial_point_fn(model=model, return_transformed=True)(0)
    
        trace_vars = {
            var.name: var
            for var in model.value_vars + model.free_RVs + model.deterministics
            if var not in model.observed_RVs + model.potentials
        }
    
>       shape_func = pytensor.compile.function.function(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
            inputs=[],
            outputs=[var.shape for var in trace_vars.values()],
            givens=(
                [(obs, model.rvs_to_values[obs]) for obs in model.observed_RVs]
                + [
                    (trace_vars[name], point[name])
                    for name in trace_vars.keys()
                    if name in point
                ]
            ),
            mode=pytensor.compile.mode.FAST_COMPILE,
            on_unused_input="ignore",
        )
E       AttributeError: 'function' object has no attribute 'function'

.pixi/envs/dev/lib/python3.12/site-packages/nutpie/compile_pymc.py:573: AttributeError
```

It seems to be an error on the PyMC side?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pixi-env: ["py311", "py312", "py313"]
+        pixi-env: ["py312", "py313"]
         pytensor-linker: ["cvm", "numba"]
 
     name: CI on ${{ matrix.pixi-env }} | ${{ matrix.pytensor-linker }}

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,7 @@ dependencies = [
     "graphviz",
     "matplotlib>=3.9.0",
     "pandas>=1.0.0",
-    "pytensor @ git+https://github.com/pymc-devs/pytensor.git@v3",  
+    "pytensor @ git+https://github.com/pymc-devs/pytensor.git",  
     "pymc @ git+https://github.com/pymc-devs/pymc.git@v6",
     "seaborn>=0.13.2,<0.14",
     "sparse>=0.17.0,<0.18",

